rustls/src/client/client_conn.rs

```diff
@@ -155,7 +160,7 @@ pub struct ClientConfig {
     /// effect.
     ///
     /// The default is true.
-    pub enable_tickets: bool,
+    pub(crate) enable_tickets: bool,
```

Comment:
If it turns out we prefer this branch's approach opposed to #1235 I think it would be sensible to adopt the https://github.com/rustls/rustls/pull/1235/commits/b5ca60b708e54f69796011dc1f94cb373f43fe37 commit that removed this field in favour of optionality for the session cache. Reading the two side-by-side I agree with @djc that having a separate bool for tickets that relies on the consumer ensuring there's a cache when the bool is set true is a little bit clumsy.